.github/workflows/ci.yml

```diff
@@ -70,14 +70,17 @@ jobs:
 
   test:
     name: Test
+    strategy:
+      matrix:
+        node-version: [18, 20]
```

Comment:
That's good thinking, let's add 22 as well while we're at it.

Reply:
added `22` and `latest` for future proofing